transmogrifier/sources/zenodo.py

```diff
@@ -18,3 +20,23 @@ def get_source_record_id(cls, xml: Tag) -> str:
                 oai_datacite XML.
         """
         return xml.header.find("identifier").string.replace("oai:zenodo.org:", "")
+
+    @classmethod
+    def valid_content_types(cls, content_type_list: List[str]) -> bool:
+        """
+        Validate a list content_type values from a Datacite XML record.
+
+        Overrides the base Datacite.valid_content_types() method.
+
+        Args:
+            content_type_list: A list of content_type values.
+        """
+        if content_type_list[0] in [
```

Comment:
This assumes that the `content_type_list` only has one value. If that ever changes in the parent class, this method will be incorrect and is likely to silently provide the wrong result. I would instead use the same strategy here as in the WHOAS version of this method, which will hedge against that future possibility.